LICENSE.md

```diff
@@ -72,8 +72,12 @@ The following components of Julia's standard library have separate licenses:
 - base/sparse/cholmod.jl (see [SUITESPARSE](http://faculty.cse.tamu.edu/davis/suitesparse.html))
 - base/special/exp.jl (see [FDLIBM](http://www.netlib.org/fdlibm/e_exp.c) [Freely distributable with preserved copyright notice])
 - base/special/rem_pio2.jl (see [FDLIBM](http://www.netlib.org/fdlibm/e_rem_pio2.c) [Freely distributable with preserved copyright notice])
+<<<<<<< HEAD
```

Comment:
⚠️

Reply:
Bah, Atom's git(hub) plugin did not agree with the way I clicked it's buttons :) Also had some extra <<<<'s in one of the other PRs